front/src/modules/stdcmAllowances/components/STDCMAllowances.tsx

```diff
@@ -49,7 +49,7 @@ const STDCMAllowances = () => {
   };
 
   return (
-    <div className="row osrd-config-item mb-2 osrd-config-item-container">
+    <div className="d-flex my-2 p-2 align-items-center osrd-config-item-container">
```

Comment:
```suggestion
    <div className="d-flex mb-2 align-items-center osrd-config-item-container">
```
padding seems useless here, as the full vertical margin

you could also change the vertical margin the same way in Itinerary.tsx